stacker/blueprints/variables/types.py

```diff
@@ -42,22 +35,17 @@ def create(self, value):
                 type
 
         """
-        new_type = self.type
-        is_list = isinstance(self.type, list)
-        if is_list:
-            new_type = self.type[0]
-        else:
-            if len(value) > 1:
-                message = (
-                    "Only one resource can be provided for single "
-                    "TroposphereType"
-                )
-                raise ValueError(message)
-
-        output = [new_type.from_dict(title, values) for title, values in
+        if not self._many and len(value) > 1:
+            message = (
+                "Only one resource can be provided for single "
```

Comment:
Is there anyway we can make this error more helpful in determining where the issue is?  Maybe we need to catch this higher and point out what caused this?